### PR TITLE
IDEMPIERE-4650 Problems with Language Management in PO

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-4650

### CODE REVIEW

**Comment by hengsin** [comment]
Do we have a page that explains how base language, tenant language and translation suppose to work ? The translation related code is very old now and I found it hard to follow what's going on in the code.

**Comment by CarlosRuiz-globalqss** [comment]
Hi @hengsin - no documentation page.

I did a bit of research for the meeting yesterday and I tried to summarize my findings there:
https://mattermost.idempiere.org/idempiere/pl/gwroaita578rxbsawpjxajgizo

Testing that is where I found the NPE on IDEMPIERE-4649 and also the inconsistent behavior being solved on this pull request.

Regards,

Carlos Ruiz

**Comment by hengsin** [comment]
I can't review this as I'm not sure what's the correct/expected behaviour here. If you are comfortable and sure with the changes, please push it through.

**Comment by CarlosRuiz-globalqss** [comment]
Thanks @hengsin - I think is my fault - I must have explained better what is the problem, I'll do it in the ticket now.

**Review by hengsin — APPROVED** [review]
good to go base on discussions on ticket.
